console/src/components/Services/Data/TableModify/ModifyActions.js

```diff
@@ -164,6 +165,138 @@ const resetPrimaryKeys = () => ({
   type: RESET_PRIMARY_KEY,
 });
 
+export const saveComputedField = (
+  computedField,
+  table,
+  originalComputedField,
+  successCb
+) => (dispatch, getState) => {
+  const migrationUp = [];
+  const migrationDown = [];
+
+  if (originalComputedField) {
+    migrationUp.push({
```

Comment:
Move all the queries to `v1QueryUtils.js`

Reply:
Done